spec/openapi.yaml

```diff
@@ -2257,6 +2257,66 @@ paths:
           "file=@filename.properties" -F 'fileUri=filename.properties'
           'https://api.smartling.com/files-api/v2/projects/$smartlingProjectId/locales/$smartlingLocaleId/file/get-translations'
 
+  '/published-files-api/v2/projects/{projectId}/files/list/recently-published':
+    get:
+      summary: List recently published files
+      description: >
+        Response is limited to files published after 14 days from the time called.
+      tags:
+      - Files
+      operationId: getRecentlyPublishedFilesList
+
+      parameters:
+      - description: A unique project identifier. This can be found in the Smartling Dashboard under Account Settings > API.
+        in: path
+        name: projectId
+        required: true
+        schema:
+          format: uuid
+          type: string
+      - description:
+          Limits the response to only those files that have a ```publishDate```
+          date after the ```publishedAfter``` parameter. The items array can
+          be empty if no files match this criteria.
+        in: query
+        name: publishedAfter
+        required: true
+        schema:
+          format: 'YYYY-MM-DDThh:mm:ssZ'
```

Comment:
I don't see a description for `fileUri` parameter

Reply:
Hmmm, this API accepts inly publishedAfter param and projectId (path param), why do you think we should describe fileUri here?
https://github.com/Smartling/published-files-service/blob/master/routes/recentlyPublishedFiles.js#L46

Reply:
never mind... my mistake